Scenario Based Samples/Save-and-SaveAs-Filter-View/README.md

```diff
@@ -0,0 +1,225 @@
+ 
+  * To run the application, use the command `dotnet run` in the terminal. After the application has started, it will display a URL in the `command line interface`, typically something like (e.g., https://localhost:5001). Copy this URL and paste it into your default web browser.
+
+    ![dashboard image](https://github.com/boldbi/aspnet-core-sample/assets/91586758/4af68f49-ffc0-400a-a323-55a3f3600a1d)
+
+> **NOTE:** We represent the dashboard embedding by default without the dashboards listing sidebar. You must navigate to the `dashboardlisting` URL (such as https://localhost:5001/dashboardlisting) to enable the dashboards list.
+
+Please refer to the [help documentation](https://help.boldbi.com/embedding-options/embedding-sdk/samples/asp-net-core/#how-to-run-the-sample) to know how to run the sample.
+
+## How to create own filter view dashboard in Javascript embedding
+
+In Javascript-based embedding, there are methods for saving, modifying, updating, retrieving and deleting filter views actions. With the help of those method, design your dashboard and bind action to your UI. So, this section explains how to create your dashboard with filter views with the necessary steps to use in your dashboard embedding.
+
+>**NOTE**: Please refer the [link](https://help.boldbi.com/managing-resources/manage-dashboards/manage-dashboard-views/) for know more about the Dashboard Views.
+
+## Steps to follow for customizing the dashboard using Views API in JavaScript embedding
+
+1. Once configured the embedConfig.JSON file in the designated location, then run your application.
+
+2. Enable the `Save`, `Save As`, and `View Saved Filters` options in the Filter Overview panel of the dashboard. To do this, set the following API to `true` in BoldBI create method:
+
+    <table style="width: 60%; float: left;">
+    <thead>
+    <tr>
+    <th style="width: 20%;">API</th>
+    <th style="width: 40%;">Description</th>
+    </tr>
+    </thead>
+    <tr>
+    <td><a href="">showSaveIcon</a></td>
+    <td>Specifies whether to show or hide the Save icon in filter overview.</td>
+    </tr>
+    <tr>
+    <td><a href="">showSaveAsIcon</a></td>
+    <td>Specifies whether to show or hide the Save As icon in filter overview.</td>
+    </tr>
+    <tr>
+    <td><a href="">showViewSavedFilterIcon</a></td>
+    <td>Specifies whether to show or hide the View Saved Filters icon in filter overview.</td>
+    </tr>
+    </table>
+
+    ![save-saveas-viewsavedfilters-API](https://github.com/boldbi/samples/assets/92368448/af824a38-6ef2-4e62-82c3-450ccd836b89)
+
+    ![enable-save-saveas-viewsavedfilter-icon](https://github.com/boldbi/samples/assets/92368448/40deb9e1-1a4a-4c93-99d7-4696cb23b4ad)
+
+### Save Filter Views
+  1. To open the `Save View` dialog, use the [onSaveFilter]() event in BoldBI create method as follows:
+
+      ![onsavefilter-event](https://github.com/boldbi/samples/assets/92368448/5609255f-1cbe-4b32-a2c2-a3650515f32e)
```

Comment:
Purpose of storing args.data in the QueryData variable?

Reply:
- Removed the console log from the code and updated the image with the changes. 
- Renamed the method name with proper understandable.
- The main objective of the queryData is to globally store the data from events (save and save as) so that the API can access these values effectively.  However, I have recently made a change to enhance the approach. Instead of using a separate variable, I now directly store the data in the data attributes of the input element.